nn/normalization.py

```diff
@@ -183,3 +184,35 @@ def normalize(a: nn.Tensor, *, axis: Union[nn.Dim, Sequence[nn.Dim]], epsilon: f
 
   mean, variance = nn.moments(a, axis=axis)
   return (a - mean) * nn.rsqrt(variance + epsilon)
+
+
+class Normalize(nn.Module):
+  """
+  :func:`normalize` with additional scale and bias
+  """
+
+  def __init__(self,
```

Comment:
I think all the args should be named args.
```suggestion
  def __init__(self, *,
```